File: Project_Api/Entities/AccountStatement.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Project_Api.Entities
{
    public class AccountStatement
    {
        [Key]
        public int Id { get; set; }
        public int SavingsAccountId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public decimal InitialBalance { get; set; }
        public decimal FinalBalance { get; set; }

    }
}

File: FrontEnd/src/models/AccountStatement.ts
export default class AccountStatement{
    id: string;
    startDate: Date;
    endDate: Date;
    initialBalance: number;
    finalBalance: number;
    minBalance: number;
    AtmOps: number;
    HumOps: number;

    constructor(id: string,startDate: Date, endDate: Date, initialBalance: number, finalBalance: number, minBalance: number, AtmOps: number, HumOps: number ){
        this.id = id;
        this.startDate = startDate;
        this.endDate = endDate;
        this.initialBalance = initialBalance;
        this.finalBalance = finalBalance;
        this.minBalance = minBalance;
        this.AtmOps = AtmOps;
        this.HumOps = HumOps;
    }
}